src/core/dynamic_batch_scheduler.cc

```diff
@@ -372,6 +373,7 @@ DynamicBatchScheduler::SchedulerThread(
             for (auto& request : requests) {
               completion_queue_.emplace_back();
               auto queue_slot = &completion_queue_.back();
+              queue_slot->second = 0;
```

Comment:
I think this two lines are not needed

Reply:
Yes, just saw this. It failed to build. I have removed the line and restarted the CI.